### DISCUSSION TITLE
No reasons for Rejected status

### DISCUSSION THREAD

**Discussion by h4s10** [community/community#120961] [GitHub Education]
### Hi everyone,

I'd seen that GitHub offer free access for students and I've canceled my Copilot trial because of this. As a consequence, I've tried 6 times to pass verification. And there are two errors, first tells me that I need to take a photo from my device, after that when I take photo trought my device I have not got any solutions, only:

> Unfortunately, we weren't able to approve your educational discount request for [@h4s10](https://github.com/h4s10).
> 
> 
> What happened?
> Common verification problems and suggested solutions are documented here: [Why wasn't my application approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved).

I need help with my verification. (Also our University doesn't have emails for students)

**Comment by abdimk** [comment]
Do you have the university id or sleep(piece of paper that has your name , semester , the course you will be taking during that semester ) or any paper like clearance with the name,date and organization name that proofs that your a student will help!

I verified mine using that piece of paper that will be given on each semester since my uv id has expired(i eventually renewed  it )

**Reply by h4s10** [reply]
Ofc I have, they decline it for some reasons(

**Comment by haululazkiyaa** [comment]
You can try to use campus ID, but the card should have a valid year.. so github can verify your identity

**Reply by h4s10** [reply]
I uploaded my student ID with valid year (2023) 6 times. Also I translated my student ID on English because my student ID is Russian and attach image that represent translation sticked with original ID.

**Reply by haululazkiyaa** [reply]
i think they decline it because your id has expired in 2023... you shoud verify with student id that expire before you apply

**Reply by h4s10** [reply]
why you decide that the 2023 is the end of the education period?

**Reply by haululazkiyaa** [reply]
sorry, did you mention "valid year (2023)" in your recent answer?

**Reply by h4s10** [reply]
Yes, I did. 2023 is a valid year. Not 'valid until'.

**Reply by haululazkiyaa** [reply]
Oh sorry.... If so, the card says it is valid until what year?? 

because what GitHub checks is the **valid until date**, then the **GitHub name must be the same as the name on the card.** It **will be stronger** if you **add your campus email** to your profile settings.

**Reply by h4s10** [reply]
All my friends have ID that represents only date of issue and they successfully pass a validation with pictures like mine.
In addition, in Russia all student IDs doesn't have end date because the university does not allow you to have a student ID in your possession if the student is no longer studying.

> It will be stronger if you add your campus email to your profile settings.

And please, read the topic message completely.

> Also our University doesn't have emails for students

**Reply by haululazkiyaa** [reply]
If so, I think there is no other way but you try with documents other than the id card, because chances are that your id will be rejected again. You can use some alternative documents such as images...

![Screenshot_2024-04-28-14-49-04-946_com android chrome](https://github.com/community/community/assets/105899777/db219767-513f-45c3-9d46-38db22b5df79)

**Reply by h4s10** [reply]
Yep, I should agree with you. All that remains is to guess, because... For some reason, Github writes the reason for the refusal in a very abstract way.

**Comment by samus-aran** [comment]
Hi @h4s10 👋 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. 

I have the following suggestions:

- It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, **so we request students to submit a live photo of their evidence rather than an upload.**
- If you are using a VPN, turn this off.

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Reply by h4s10** [reply]
Did you mean that I need submit (upload) a photo via my device, for example laptop?
If so, I wrote in the topic post next:

> And there are two errors, first tells me that I need to take a photo from my device, after that when I take photo trought my device I have not got any solutions
